apps/examples/security_test/seclink/sl_test_utils.c

```diff
@@ -64,9 +64,12 @@ int sl_test_malloc_buffer(hal_data *data, int buf_len)
 
 int sl_test_malloc_buffer_priv(hal_data *data, int buf_len)
 {
+	if (!data) {
+		return -1;
+	}
 	data->priv = (unsigned char *)zalloc(buf_len);
 	if (!data->priv) {
-		return -1;
+		return -2;
```

Comment:
What is the meaning of `-1` and `-2`?
Just for checking where to fail?

Reply:
Even if my assumption is correct, I don't know why we need this. Because we don't use/check the return value in anywhere.

Reply:
in sl_key_test they check only success return, so just for future checking I have added

Reply:
Okay. Thanks.